project/src/scraper/palacio.py

```diff
@@ -0,0 +1,204 @@
+from __future__ import annotations
+
+import random
+import time
+from dataclasses import dataclass
+
+from bs4 import BeautifulSoup
+from bs4.element import Tag
+from selenium import webdriver
+from selenium.common.exceptions import (
+    ElementNotInteractableException,
+    NoSuchElementException,
+)
+from selenium.webdriver.common.by import By
+
+from .base import BaseScraper
+from .utils import normalize_string
+
+BASE_URL = "https://www.elpalaciodehierro.com"
+PRODUCT_URL = f"{BASE_URL}/buscar?q=lavadoras"
+
+
+@dataclass
+class ParentScraper(BaseScraper):
+    driver: webdriver.Firefox
+
+    @property
+    def service_name(self) -> str:
+        return "Palacio de hierro"
```

Comment:
Consiodera utilizar string estandarizado
 
```suggestion
        return "palacio_hierro"
```